### ISSUE TITLE
GRIDSS wall time improvements

### ISSUE THREAD

**Issue by d-cameron** [hartwigmedical/pipeline5#150]
GRIDSS now has mult-node support so the wall clock time can be reduced considerably. The two operations parallelisable across multiple nodes are:

- Preprocessing. With preprocessing now using a JNI version of bwa, preprocessing now utilises 8 cores pretty well. Preprocessing can be done separately for each input BAM. This gives us a ~35% improvement in preprocessing time as T and N can be processed in parallel (T being the bottleneck)

- Assembly .This step is over 50% of GRIDSS runtime. The `gridss.sh` script supports parrallelisation of assembly across multiple nodes using the `--jobindex` and `--jobnodes` parameters. Each node does the (n+i)th chunk. After the jobs are complete, an assemble gather task is needed to merge the per-chunk output files together (just copy in all the intermediate files to their expect locations and run `gridss.sh` with `-s all` and it'll finish everything, including the variant calling and annotation).

We'll need to benchmark the overhead of copying the data and setting up an instance.

COLO829 hg38 runtime timing breakdown on a Ryzen 3700X (8/16 @ 4.1Ghz)
Preprocessing: 30m N + 1h14m T
Assembly: 2h33m
Variant calling & Annotation: 47m

TLDR: preprocess job per input file, assembly across n nodes, and a final job to produce the VCF.
